polynote-frontend/polynote/ui/util/tags.ts

```diff
@@ -107,6 +107,19 @@ export function icon(classes: string[], iconName: string, alt?: string) {
     return span(classes, img(['icon'], `/style/icons/fa/${iconName}.svg`, alt || iconName))
 }
 
+export function a(classes: string[], href: string, contentOrPreventNavigate: Content | boolean, contentWhenPreventNavigate?: Content) {
```

Comment:
why not `a(..., content: Content, preventNavigate=false)`?

Reply:
I dunno, just thought it would be annoying that `content` is usually last but not in this case? I don't mind changing it if you think it's weird... I think this is the norm in Python but I don't really know what the conventions are in typescript (or python really for that matter 😄 )

Reply:
Hmm, would `a(..., preventNavigate=false, content?: Content)` work?